lib/improver/feels_like_temperature.py

```diff
@@ -0,0 +1,240 @@
+        Cube of relative humidities (fractional)
+
+      pressure (iris.cube.Cube):
+        Cube of air pressure (Pa)
+
+    Returns:
+      apparent_temperature (iris.cube.Cube):
+        Cube of apparent temperatures (K)
+
+    References:
+      Steadman, R. (1984). A Universal Scale of Apparent Temperature.
+      Journal of Climate and Applied Meteorology, 23(12), pp.1674-1687
+    """
+    # take a copy of each cube's orignial units
+    temp_units = temperature.copy().units
+    wind_units = wind_speed.copy().units
+    pressure_units = pressure.copy().units
+    relative_humidity_units = relative_humidity.copy().units
+
+    # ensure units are correct
+    wind_speed.convert_units('m s-1')
+    pressure.convert_units('Pa')
+    relative_humidity.convert_units('1')
+    temperature.convert_units('K')
+    # look up saturated vapour pressure
+    svp = WetBulbTemperature().lookup_svp(temperature)
+    # convert to SVP in air
+    svp = WetBulbTemperature().pressure_correct_svp(
+        svp, temperature, pressure)
+    # convert temperature units
+    temperature.convert_units('celsius')
+    # calculate actual vapour pressure
+    # and convert relative humidities to fractional values
+    avp_data = svp.data*relative_humidity.data
+    avp = svp.copy(avp_data)
+    avp.rename("actual_vapour_pressure")
+    avp.convert_units('kPa')
+    # calculate apparent temperature
+    apparent_temperature_data = (
+        -2.7 + 1.04*temperature.data + 2.0*avp.data
+        - 0.65*wind_speed.data)
+    apparent_temperature = temperature.copy(data=apparent_temperature_data)
+    apparent_temperature.rename("apparent_temperature")
+    apparent_temperature.convert_units('K')
+
+    # convert units back to input units
+    temperature.convert_units(temp_units)
+    wind_speed.convert_units(wind_units)
+    pressure.convert_units(pressure_units)
+    relative_humidity.convert_units(relative_humidity_units)
+
+    return apparent_temperature
+
+
+def calculate_feels_like_temperature(temperature, wind_speed,
+                                     relative_humidity, pressure):
+    """
+    Calculates the feels like temperature using a combination of
+    the wind chill index and Steadman's apparent temperature equation with
+    the following method:
+
+    If temperature < 10 degress C: The feels like temperature is equal to
+    the wind chill.
+
+    If temperature > 20 degress C: The feels like temperature is equal to
+    the apparent temperature.
+
+    If 10 <= temperature <= 20 degrees C: A weighting (alpha) is calculated
+    in order to blend between the wind chill and the apparent temperature.
+
+    Args:
+      temperature (iris.cube.Cube):
+        Cube of air temperatures (K)
+
+      wind_speed (iris.cube.Cube):
+        Cube of 10m wind speeds (m/s)
+
+      relative_humidity (iris.cube.Cube):
+        Cube of relative humidities (fractional)
+
+      pressure (iris.cube.Cube):
+        Cube of air pressure (Pa)
```

Comment:
Because the function converts the units on input, there is no need to specify what the argument units should be. This is true for the other two functions too.

Reply:
Removed these and changed the description of the units of the output cubes.